client/client.go

```diff
@@ -146,6 +146,16 @@ func NewClient(baseDir string) (*Client, error) {
 		return nil, fmt.Errorf("error reading CT file: %v", err)
 	}
 
+	// Create the initial sync file.
+	if err := c.updateSyncFile(); err != nil {
+		panic("could not create sync file: " + fmt.Sprintf("%v", err))
+	}
+
+	// Create the initial report file.
+	if err := c.updateReportFile(); err != nil {
+		panic("could not create sync file: " + fmt.Sprintf("%v", err))
```

Comment:
same comment as above, return an error and let the caller decide what to do with it